jams/core.py

```diff
@@ -1016,10 +892,10 @@ def trim(self, start_time, end_time, strict=False):
         # We do this rather than copying and directly manipulating the
         # annotation' data frame (which might be faster) since this way trim is
         # independent of the internal data representation.
-        for idx, obs in self.data.iterrows():
+        for obs in self.data.obs:
```

Comment:
I *would* be keen to simplify this and make the object directly iterable, by a subclass or whatever